### DISCUSSION TITLE
How the word Zustand is pronounced?

### DISCUSSION THREAD

**Discussion by indicozy** [pmndrs/zustand#1442] [Q&A] [answered]
Just a quick question, is it like Zoostand or zAstand?

**Comment by dai-shi** [comment, marked as answer]
If you want to know the pronunciation in German, google translate can help.
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/490574/203187901-0a41c6b0-74eb-49a3-a4c5-0b644f5d92ff.png">

**Reply by indicozy** [reply]
Thanks!
